src/components/background/classes.js

```diff
@@ -20,7 +20,7 @@ function BackgroundClasses( attributes ) {
 		{ 'has-background': attributes.backgroundColor || attributes.customBackgroundColor },
 		{ [ backgroundClass ]: backgroundClass },
 		{ 'has-parallax': attributes.backgroundImg && attributes.hasParallax },
-		{ 'has-background-image': attributes.backgroundImg },
+		{ [ `has-background-${ attributes.backgroundType }` ] : attributes.backgroundImg && attributes.backgroundType },
```

Comment:
@phpbits Will this cause deprecation issues for folks running older versions of CoBlocks, before `backgroundType` existed?

Reply:
@richtabor I've actually changed this to `has-background-${ attributes.backgroundType }` 👍